tests/strands/agent/test_agent.py

```diff
@@ -1010,8 +1010,14 @@ def test_agent_init_with_trace_attributes():
     assert "invalid_nested_list" not in agent.trace_attributes
 
 
+@pytest.fixture
+def mock_metrics_client():
+    with unittest.mock.patch("strands.agent.agent.MetricsClient") as mock_metrics_client:
+        yield mock_metrics_client
+
+
 @unittest.mock.patch("strands.agent.agent.get_tracer")
-def test_agent_init_initializes_tracer(mock_get_tracer):
+def test_agent_init_initializes_tracer(mock_get_tracer, mock_metrics_client):
```

Comment:
Given that none of the tests are using the metric_client, what's the need for including it each test?